subsys/bluetooth/host/l2cap.c

```diff
@@ -539,7 +540,7 @@ static int l2cap_ecred_conn_req(struct bt_l2cap_chan **chan, int channels)
 	req->psm = sys_cpu_to_le16(ch->psm);
 	req->mtu = sys_cpu_to_le16(ch->rx.mtu);
 	req->mps = sys_cpu_to_le16(ch->rx.mps);
-	req->credits = sys_cpu_to_le16(ch->rx.init_credits);
```

Comment:
@alwa-nordic What do we do with `init_credits` now? I suggest we keep it and just mark it as internal-use in l2cap.h. 
This way we don't break the build of existing apps.

Reply:
I agree.